scripts/lib/logger/__init__.py

```diff
@@ -45,7 +45,10 @@ def format(self, record):
     prefix = '\n' if ConsoleFormatter.lineCount > 0 else ''
     if record.levelname == 'ERROR' or record.levelname == 'WARNING':
       msg = '{}[{}]: {}'.format(prefix, record.levelname, msg)
-    elif record.levelname != 'COMPLETED':
+    elif record.levelname == 'COMPLETED':
+      suffix = '\n' if ConsoleFormatter.lineCount > 1 else ''
+      msg = msg + suffix
+    else:
```

Comment:
We are using line count. Wouldn't this always cause completed after the first ones to be in a new line ?
Does the count track the number of lines logged ?